package.json

```diff
@@ -325,6 +325,7 @@
     "electron-debug": "^3.2.0",
     "electron-store": "^8.1.0",
     "electron-updater": "^6.1.5",
+    "fuse.js": "^6.6.2",
```

Comment:
extra?

Reply:
Yes. The issue mentioned that I could use an npm package to solve it. So I used this for efficient working!

Reply:
I don't see it used (imported) in the code anywhere. Am I missing something?

Reply:
Ahh.... Forgot to remove that package...i again found that the algorithm i coded worked so i replaced fuse with that...i forgot to update the PR and remove the package... Will do it now!!! 

Thanks